src/containers/forms/LogActivityForm.jsx

```diff
@@ -180,7 +180,7 @@ class LogActivityForm extends Component {
   resetState = () => {
     this.setState({
       activityTypeId: '',
-      numberOf: '',
+      noOfParticipants: '',
```

Comment:
Not sure about this convention. I prefer full names i.e `numberOfParticipants`.

Reply:
I could use full names for the state but leave the payload sent to the backend to have `noOfParticipants`. I matched the state to the response object from the backend.